src/animation_commands_extra.c

```diff
@@ -15,6 +15,182 @@ extern u8 gVramGraphicsCopyQueueIndex;
 extern struct GraphicsData *gVramGraphicsCopyQueue[];
 
 extern const AnimationCommandFunc animCmdTable[];
+extern const AnimationCommandFunc animCmdTable_2[];
+
+#define ReadInstruction(script, cursor) ((void *)(script) + (cursor * sizeof(s32)))
+
+#if 1
```

Comment:
Also this doesn't need to be here anymore right?

Reply:
Sure, I guess? If you find a better place for it, go ahead! :D